client/src/pages/Profile.jsx

```diff
@@ -82,7 +83,7 @@ const Profile = () => {
         return;
       }
 
-      if (!newUser) {
+      if (!newUser && memberDataResponse !== null) {
```

Comment:
`&& memberDataResponse) {...}` is more succinct.

Reply:
Again, here being wary of the sketchy-null check